src/include/svrfunc.h

```diff
@@ -54,6 +54,25 @@ extern "C" {
 #include "pbs_sched.h"
 #include "pbs_entlim.h"
 
+/*
+ * Convert given subjob index to tracking table array index
+ *
+ * For example: -J10-20:7 creates 2 subjob
+ *   1. [10] on 0th index in table
+ *   2. [17] on 1st index in table
+ * So SJ_IDX_2_TBLIDX(pj, 17) == ((17 - 10) / 7) == 1
+ */
+#define SJ_IDX_2_TBLIDX(pj, sjidx) ((int)((sjidx - pj->ji_ajtrk->tkm_start) / pj->ji_ajtrk->tkm_step))
+
```

Comment:
A downside to this over searching is you can get a false positive.  If I search for subjob index 15, I won't find it.  If I do the above math, I'll find subjob 17.

Reply:
Good catch @bhroam, I should have thought of negative case... will fix it.